package.json

```diff
@@ -47,6 +47,7 @@
     "@ngrx/store": "^9.2.0",
     "@ngrx/store-devtools": "^9.2.0",
     "@ngx-translate/core": "^13.0.0",
+    "cropperjs": "1.5.11",
```

Comment:
Have you created a PR for ADW as well? Merging this PR might make ADW failed, if it doesn't have the same packages.

Reply:
gonna do that now, thanks.